### PR TITLE
Release 3.0: Update FAQ articles mentioning "Warn for others"

### PR DESCRIPTION
**Pull request by larswmh**
This PR updates "Warn for others" FAQs in response to changes regarding TAN procedures in 3.0.

Dates will be inserted once approved or shortly before being merged. Apart from that, this PR is ready to be reviewed.

[Kann ich die Stellvertreter-Funktion nutzen, egal welchen QR-Code ich eingescannt habe?](https://www.coronawarn.app/de/faq/results/#check_in_luca_warn_for_others)
image
[Can I warn for others regardless of which QR code I have scanned?](https://www.coronawarn.app/en/faq/results/#check_in_luca_warn_for_others)
image

[Wie kann ich in Vertretung warnen?](https://www.coronawarn.app/de/faq/results/#warn_for_others)
image
[How can I warn for others?](https://www.coronawarn.app/en/faq/results/#warn_for_others)
image

---
Internal Tracking ID: [EXPOSUREAPP-14499](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14499)

### CODE REVIEW

**Comment by MikeMcC399** [comment]
This was not changed in this PR, however the Blogs use a "Gender-Stern" not a colon, so maybe this should be standardized in the FAQs? i.e. "Veranstalter*innen" instead of "Veranstalter:innen".
It seems that there is no DIN standard on this, but at least this website should be internally consistent.

File: src/data/faq.json [hunk 1]
```diff
@@ -354,7 +354,10 @@
                                 "anchor": "warn_for_others",
                                 "active": false,
                                 "textblock": [
-                                    "As an organizer of an event, starting with version 2.9, you can warn for others, such as for a person who has tested positive and was attending an event but not checked in via the Corona-Warn-App (CWA), upon request by the health department.",
+                                    "<b>Updated ?????</b>",
+                                    "This feature of the Corona-Warn-App is no longer available from version 3.0 and cannot be used with the previous versions 2.9 to 2.28 after ??????, as the verification hotline (TAN hotline) will be discontinued on this date.",
```
Comment (MikeMcC399) [inline src/data/faq.json:358]: from version 3.0 onwards
or
in version 3.0 and later versions

File: src/data/faq.json [hunk 2]
```diff
@@ -354,7 +354,10 @@
                                 "anchor": "warn_for_others",
                                 "active": false,
                                 "textblock": [
-                                    "As an organizer of an event, starting with version 2.9, you can warn for others, such as for a person who has tested positive and was attending an event but not checked in via the Corona-Warn-App (CWA), upon request by the health department.",
+                                    "<b>Updated ?????</b>",
+                                    "This feature of the Corona-Warn-App is no longer available from version 3.0 and cannot be used with the previous versions 2.9 to 2.28 after ??????, as the verification hotline (TAN hotline) will be discontinued on this date.",
+                                    "<hr>",
+                                    "As an organizer of an event, starting with version 2.9 and up to version 2.28 of the Corona-Warn-App (through ??????), you can warn for others, such as for a person who has tested positive and was attending an event but not checked in via the Corona-Warn-App (CWA), upon request by the health department.",
```
Comment (MikeMcC399) [inline src/data/faq.json:360]: (through ??????) =>

if this is a single date, then

(until ??????)

File: src/data/faq.json [hunk 3]
```diff
@@ -1186,6 +1189,9 @@
                                 "anchor": "check_in_luca_warn_for_others",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated ?????</b>",
+                                    "This feature of the Corona-Warn-App is no longer available from version 3.0 and cannot be used with the previous versions 2.9 to 2.28 after ??????, as the verification hotline (TAN hotline) will be discontinued on this date.",
```
Comment (MikeMcC399) [inline src/data/faq.json:1193]: same comment about "from"

Comment (larswmh) [reply src/data/faq.json]: Changed in https://github.com/corona-warn-app/cwa-website/pull/3312/commits/4ffe8f2c1cf64712dae8d52554bf516c25027efa, thanks!

File: src/data/faq.json (hunk 2 above)
Comment (larswmh) [reply src/data/faq.json]: Changed in https://github.com/corona-warn-app/cwa-website/pull/3312/commits/4ffe8f2c1cf64712dae8d52554bf516c25027efa, thanks!

File: src/data/faq.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq.json]: Changed in https://github.com/corona-warn-app/cwa-website/pull/3312/commits/4ffe8f2c1cf64712dae8d52554bf516c25027efa, thanks!

**Comment by larswmh** [comment]
@MikeMcC399 

Your suggested changes have been made in the most recent commit. As this was the only case where we would use `:innen` on the site, I've changed it to `*innen` which has many occurences on the site already.

**Review by dsarkar — APPROVED** [review]

**Pull request merged**